stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/header/MessageListHeader.kt

```diff
@@ -23,12 +23,12 @@ import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.style.TextOverflow
 import androidx.compose.ui.unit.dp
+import com.getstream.sdk.chat.state.MessageMode
+import com.getstream.sdk.chat.state.Normal
+import com.getstream.sdk.chat.state.Thread
```

Comment:
Any chance we can rename `Thread` at some point? It's very hard to import this, as `java.lang.Thread` is imported for Kotlin users by default.

Reply:
Yup, it forces manual import. Maybe it could be a `TreadReply`?  cc @filbabic